sdk/loadtestservice/Azure.Developer.LoadTesting/Hero/Program.cs

```diff
@@ -0,0 +1,200 @@
+            string testId = "my-dotnet-test-id";
+            string testRunId = "my-dotnet-test-run-id";
+            string fileName = "my-dotnet-file-name.jmx";
+            string subscriptionId = Environment.GetEnvironmentVariable("SUBSCRIPTION_ID");
+            string appComponentConnectionString = "/subscriptions/" + subscriptionId + "/resourceGroups/App-Service-Sample-Demo-rg/providers/Microsoft.Web/sites/App-Service-Sample-Demo";
+
+            // create loadtest
+            try
+            {
+                Response response = loadTestAdministrationClient.CreateOrUpdateTest(testId, RequestContent.Create(
+                   new
+                   {
+                       description = "This is created using SDK",
+                       displayName = "SDK's LoadTest",
+                       loadTestConfig = new
+                       {
+                           engineInstances = 1,
+                           splitAllCSVs = false,
+                       },
+                       secrets = new { },
+                       enviornmentVariables = new { },
+                       passFailCriteria = new
+                       {
+                           passFailMetrics = new { },
+                       }
+                   }));
+
+                Console.WriteLine("Loadtest created successfully");
+                Console.WriteLine(response.Content.ToString());
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+
+
+            // upload file
+            try
+            {
+                Response response = loadTestAdministrationClient.UploadTestFile(testId, fileName, RequestContent.Create(
+                        File.OpenRead(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "sample.jmx"))
+                    ));
+
+                Console.WriteLine("File uploaded successfully");
+                Console.WriteLine(response.Content.ToString());
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+
+            // wait for file to complete Validation
+            try
+            {
+                TestFileValidationStatus testFileValidationStatus = loadTestAdministrationClient.BeginGetTestScriptValidationStatus(testId);
+
+                Console.WriteLine(testFileValidationStatus.ToString());
+
+                if(testFileValidationStatus != TestFileValidationStatus.ValidationSuccess)
+                {
+                    // if file validaion failed, then stop the process
+                    return;
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+
+            // connecting app component
+            try
+            {
+                var data = new
```

Comment:
why is creation of a data object in a try block? Did we want to create an appComponent?

Reply:
These needs correction, will do.